modules/service/data-service.js

```diff
@@ -2,27 +2,56 @@ const { v1: uuidv1 } = require('uuid');
 const constants = require('../constants');
 const GraphDB = require('../../external/graphdb-service');
 
+// TODO: Discussion on this values
+const MAX_RETRIES = 10;
```

Comment:
Move this to constants

Reply:
Just to continue on this topic, should we have 10 retries every 10 seconds, i think that is acceptable to wait 1-2 minutes for graphdb to start, and if it is not started then stop the ot-node and ask ot-node runner to check the graphdb